parser.js

```diff
@@ -637,6 +636,10 @@
           // then the slash is ignored and the number is matched later
           // as normal characters.
 
+          // Recall the negative decision to decide if the input must be parsed
+          // a second time with the total normal-groups.
+          backrefDenied.push(refIdx);
```

Comment:
If anyone comes up with a better name than `backrefDenied` I would be more than welcome for it :)